cmd/root.go

```diff
@@ -32,10 +48,34 @@ var rootCmd = &cobra.Command{
 		if verbose {
 			log.SetLevel(log.DebugLevel)
 		}
+		configureLogrusHooks()
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Error return value of `configureLogrusHooks` is not checked (errcheck)